assets/js/modules/reader-revenue-manager/datastore/constants.js

```diff
@@ -19,3 +19,9 @@
 export const MODULES_READER_REVENUE_MANAGER = 'modules/reader-revenue-manager';
 
 export const ERROR_CODE_NON_HTTPS_SITE = 'non_https_site';
+
+export const PUBLICATION_ONBOARDING_STATE = {
```

Comment:
What do you think about changing this to `PUBLICATION_ONBOARDING_STATES` (plural)?

Reply:
Given that only one state can be referenced at a time for a publication, I thought the singular form was appropriate. I can change it if you prefer the plural.

Reply:
Updated ✅